src/Microsoft.SqlTools.ServiceLayer/Agent/Contracts/AgentNotebookRequest.cs

```diff
@@ -256,4 +256,32 @@ public static readonly
             RequestType<UpdateAgentNotebookRunPinParams, ResultStatus> Type =
             RequestType<UpdateAgentNotebookRunPinParams, ResultStatus>.Create("agent/updatenotebookpin");
     }
+
+     /// <summary>
+    /// SQL Agent Notebook materialized params
+    /// </summary>
+    public class DeleteNotebookMaterializedParams : TaskRequestDetails
+    {
+        public string OwnerUri { get; set; }
+        public string TargetDatabase { get; set; }
+        public AgentNotebookHistoryInfo agentNotebookHistory { get; set; }
+    }
+
+    /// <summary>
+    /// SQL Agent Notebook materialized result
+    /// </summary>
+    public class DeleteNotebookMaterializedResult : ResultStatus
+    {
+        public string NotebookMaterialized { get; set; }
+    }
+
+    /// <summary>
+    /// SQL Agent Notebook materialized request type
+    /// </summary>
+    public class DeleteNotebookMaterializedRequest
+    {
+        public static readonly
+            RequestType<DeleteNotebookMaterializedParams, ResultStatus> Type =
+            RequestType<DeleteNotebookMaterializedParams, ResultStatus>.Create("agent/deletenotebookmaterialized");
```

Comment:
>deletenotebookmaterialized [](start = 86, length = 26)

Same here - this seems like it should probably be agent/deletematerializednotebook?